frontend/src/pages/SignIn.jsx

```diff
@@ -1,27 +1,39 @@
 import React, { useState } from "react"
+import { firebaseApp } from "../lib/firebaseApp/index.js"
+
+console.log(firebaseApp)
 
 export const SignIn = () => {
   const [email, setEmail] = useState("")
   const [password, setPassword] = useState("")
 
-  const submit = () => {
+  const submit = async () => {
     alert(`
       ${email}
       ${password}
     `)
+
+    try {
+      const res = await firebaseApp.auth().signInWithEmailAndPassword(email, password)
```

Comment:
tryを利用しない簡潔な書き方があるっぽい
https://qiita.com/akameco/items/cc73afcdb5ac5d0774bc

```suggestion
      const res = await firebaseApp.auth().signInWithEmailAndPassword(email, password).catch(error => console.log(error))
```